packages/contracts-periphery/deploy/op-nft/AttestationStationProxy.ts

```diff
@@ -16,22 +13,33 @@ const { getAddress } = utils
  */
 const deployFn: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
   const { deployer } = await hre.getNamedAccounts()
+  const ddd = hre.deployConfig.ddd
+
+  if (getAddress(deployer) !== getAddress(ddd)) {
+    throw new Error('Must deploy with the ddd')
+  }
```

Comment:
This assertion is here to ensure that the impl and admin can be set with deterministic deployment

Reply:
The `ddd` doesn't need to be the deployer. It just needs to be the owner.

Reply:
I was going for a single key deploy + config. It feels like more work to deploy with one key then use another key to config